runtime/doc/eval.txt

```diff
@@ -4031,9 +4031,11 @@ jobsend({job}, {data})					{Nvim} *jobsend()*
 			:call jobsend(j, ["abc", "123\n456", ""])
 < 		will send "abc<NL>123<NUL>456<NL>".
 
-jobstart({argv}[, {opts}])			{Nvim} *jobstart()*
-		Spawns {argv}(list) as a job. If passed, {opts} must be a
-		dictionary with any of the following keys:
+jobstart({cmd}[, {opts}])				{Nvim} *jobstart()*
+		Spawns {cmd} as a job.  If {cmd} is a |String|, it will be ran
+		via "'shell' 'shellcmdflag' {cmd}", or directly if a |List|.
```

Comment:
Perhaps remove the outer quotes here, it could be interpreted as we're concatenating a string here, but we're really building up an argument list.